packages/nextjs/app/builders/page.tsx

```diff
@@ -18,30 +17,18 @@ const BuildersPage: NextPage = () => {
       .filter((builder): builder is string => !!builder) || [];
   const uniqueBuilders = Array.from(new Set(builderAddresses));
 
-  const { resolvedTheme } = useTheme();
-  const isDarkMode = resolvedTheme === "dark";
-
   return (
-    <div
-      className={`flex flex-col items-center justify-center min-h-screen ${isDarkMode ? "bg-gray-750" : "bg-gray-200"}`}
-    >
-      <div className={`max-w-7xl mx-auto p-8 ${isDarkMode ? "bg-gray-800" : "bg-white"} shadow-2xl rounded-2xl my-10`}>
-        <h1 className={`text-4xl font-extrabold text-center ${isDarkMode ? "text-white" : "text-gray-900"} mb-10`}>
-          Our Builders
-        </h1>
+    <div className={`flex flex-col items-center justify-center min-h-screen bg-base-200`}>
+      <div className={`max-w-7xl mx-auto p-8 bg-base-300 shadow-2xl rounded-2xl my-10`}>
```

Comment:
Nitpick: Can we now just wrap them inside double quotes instead of doing `{`...`}`

Reply:
Done ✅